dashboard/test/factories/factories.rb

```diff
@@ -1146,6 +1146,22 @@
     group 1
   end
 
+  factory :regional_partner_summer_workshops do
```

Comment:
What class is this for? I don't see a `RegionalPartnerSummerWorkshops` model. Does this work? It looks like it might be intended as a trait, or child of, the regional partner factory?